coala_quickstart/generation/FileGlobs.py

```diff
@@ -28,6 +30,12 @@ def get_project_files(log_printer, printer, project_dir):
                       color="green")
         ignore_globs = get_gitignore_glob(project_dir)
 
+    if noninteractive:
+        if ignore_globs is None:
+            file = open(".gitignore", "w")
+            file.write("src")
```

Comment:
Why add `src` to the gitignore? This would ignore the source directory in some projects, which is the exact opposite of what we need!

Just create an empty file.